analyzers/src/SonarAnalyzer.CFG/ShimLayer/AnalysisContext/SymbolStartAnalysisContextWrapper.cs

```diff
@@ -52,26 +56,29 @@ namespace SonarAnalyzer.ShimLayer.AnalysisContext;
     [ExcludeFromCodeCoverage]
     static SymbolStartAnalysisContextWrapper()
     {
-        var symbolStartAnalysisContextType = typeof(CompilationStartAnalysisContext).Assembly.GetType("Microsoft.CodeAnalysis.Diagnostics.SymbolStartAnalysisContext");
+        var symbolStartAnalysisContextType = TryLoadSymbolStartAnalysisContextType();
+        var languageKindEnumVBType = TryLoadLanguageKindEnumVBType();
         CancellationTokenAccessor = CreatePropertyAccessor<CancellationToken>(nameof(CancellationToken));
         CompilationAccessor = CreatePropertyAccessor<Compilation>(nameof(Compilation));
         OptionsAccessor = CreatePropertyAccessor<AnalyzerOptions>(nameof(Options));
         SymbolAccessor = CreatePropertyAccessor<ISymbol>(nameof(Symbol));
         RegisterCodeBlockActionMethod = CreateRegistrationMethod<CodeBlockAnalysisContext>(nameof(RegisterCodeBlockAction));
         RegisterCodeBlockStartActionCS =
             CreateRegistrationMethod<CodeBlockStartAnalysisContext<CS.SyntaxKind>>(nameof(RegisterCodeBlockStartAction), typeof(CS.SyntaxKind));
```

Comment:
This and the other breaklines in this initialization should go away. It's way easier to read it as single line commands